src/_includes/layouts/homepage.html

```diff
@@ -10,13 +10,20 @@
   <body data-typeface="system-ui">
     <main data-layout="centered-single-column">
       <h1>Lunch.dev Community Calendar</h1>
+
+      <h2>Upcoming Events</h2>
```

Comment:
Loving these `h2`s, but tell ya what: we should use these `h2`s to label the `ul`s underneath them. I'm thinking something like:

```html
<h2 id="upcoming-events">Upcoming Events</h2>
<ul aria-labelledby="upcoming-events">
    // ...
</ul>
```

That way, as screenreader users navigate through the list, they'll be told stuff like "Upcoming Events, list, 4 items" and "Upcoming Events, 3 of 4".

Thoughts?

```

Reply:
excellent call!